contrib/mesos/pkg/scheduler/meta/annotations.go

```diff
@@ -25,7 +25,6 @@ const (
 	TaskIdKey                = "k8s.mesosphere.io/taskId"
 	SlaveIdKey               = "k8s.mesosphere.io/slaveId"
 	OfferIdKey               = "k8s.mesosphere.io/offerId"
-	ExecutorIdKey            = "k8s.mesosphere.io/executorId"
```

Comment:
We don't store the ExecutorInfo anymore in the podtask. Therefore, we don't have to persist anything about it using annotations anymore.